sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -3741,3 +3745,39 @@ def seal_append_blob(self, **kwargs):
             return self._client.append_blob.seal(**options) # type: ignore
         except HttpResponseError as error:
             process_storage_error(error)
+
+    @distributed_trace
+    def get_container_client(self, **kwargs):
+        # type: (...) -> ContainerClient
+        """Get a client to interact with the blob's parent container.
+
+        The container need not already exist. Defaults to current blob's credentials.
+
+        :keyword credential:
```

Comment:
@kasobol-msft @annatisch how do you feel about having credential option when get_container_client from blob client?

Reply:
I think we should keep it simple. It's a reverse operation to the `container.get_blob_client`, so it seems that this should be parameter-less/option-less ?

https://github.com/Azure/azure-sdk-for-python/blob/6286f95b28942b25da258ea45eba8c00c386f176/sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py#L1474-L1500

Reply:
btw. it's not [yagni](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)